.github/workflows/docker-integration-tests.yaml

```diff
@@ -0,0 +1,69 @@
+name: Docker images for integration tests
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+env:
+  PLATFORMS: "linux/amd64"
+
+jobs:
+  docker:
+    name: Build and push Docker image
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - dockerfile: ./integration-tests/Dockerfile_alpine3
+            image: goss_alpine3
+          - dockerfile: ./integration-tests/Dockerfile_arch
+            image: goss_arch
+          - dockerfile: ./integration-tests/Dockerfile_centos7
+            image: goss_centos7
+          - dockerfile: ./integration-tests/Dockerfile_trusty
+            image: goss_trusty
+          - dockerfile: ./integration-tests/Dockerfile_wheezy
+            image: goss_wheezy
+    permissions:
+      packages: write
+      contents: read
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ghcr.io/${{ github.repository_owner }}/${{ matrix.image }}
+
+      - name: Build and push tag
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ${{ matrix.dockerfile }}
+          push: true
+          tags: |
+            ghcr.io/${{ github.repository_owner }}/${{ matrix.image }}:latest

```

Comment:
Is the intention to move away from `latest` later on?

Reply:
This pull request currently represents the current state of the repository (images: https://hub.docker.com/u/aelsabbahy , build-script: https://github.com/goss-org/goss/blob/master/development/build_images.sh) with the perspective of updating the images more often in the future than today. In the future it will certainly need other tags than latest, e.g. hash value of the docker file or date of the build.